tests/unit/test_file_output.py

```
@@ -0,0 +1,132 @@
+	f1 = File(path='/tmp/test.txt', category='download')
+	f2 = File(path='/tmp/test.txt', category='download')
+	f3 = File(path='/tmp/other.txt', category='download')
+
+	assert f1 == f2
+	assert f1 != f3
+
+
+def test_file_to_dict():
+	"""Test File conversion to dictionary."""
+	f = File(
+		path='/tmp/test.txt',
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/test.txt"

(S108)